tests/components/switchbot_cloud/test_button.py

```diff
@@ -0,0 +1,68 @@
+"""Test for the switchbot_cloud bot as a button."""
+
+from unittest.mock import patch
+
+from switchbot_api import Device
+
+from homeassistant.components.button import DOMAIN as BUTTON_DOMAIN, SERVICE_PRESS
+from homeassistant.components.switchbot_cloud import SwitchBotAPI
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import ATTR_ENTITY_ID, STATE_UNKNOWN
+from homeassistant.core import HomeAssistant
+
+from . import configure_integration
+
+
+async def test_pressmode_bot(
+    hass: HomeAssistant, mock_list_devices, mock_get_status
+) -> None:
+    """Test press."""
+    mock_list_devices.return_value = [
+        Device(
+            deviceId="bot-id-1",
+            deviceName="bot-1",
+            deviceType="Bot",
+            hubDeviceId="test-hub-id",
+        ),
+    ]
+
+    mock_get_status.return_value = {"deviceMode": "pressMode"}
+
+    entry = configure_integration(hass)
+    await hass.config_entries.async_setup(entry.entry_id)
+    await hass.async_block_till_done()
+
+    assert entry.state is ConfigEntryState.LOADED
+
+    entity_id = "button.bot_1"
+    assert hass.states.get(entity_id).state == STATE_UNKNOWN
+    with patch.object(SwitchBotAPI, "send_command"):
+        await hass.services.async_call(
+            BUTTON_DOMAIN, SERVICE_PRESS, {ATTR_ENTITY_ID: entity_id}, blocking=True
+        )
+    assert hass.states.get(entity_id).state != STATE_UNKNOWN
+
+
+async def test_switchmode_bot(
```

Comment:
```suggestion
async def test_switchmode_bot_no_button_entity(
```
just to make it a bit clearer that this is a "negative" test

Reply:
Test name updated 👍